### ISSUE TITLE
fix: flipper didn't show up timely in horizontal-scroll

### ISSUE THREAD

**Issue by akronbook** [microsoft/fast#5386]
# 🐛 Bug Report

when minimizing the browser window, fast-buttons were hidden, but the navigation flipper control inside a horizontal-scroll didn't show up as was expected.

## 💻 Repro or Code Sample

https://github.com/akronbook/horizontal-scroll-bug

## 🤔 Expected Behavior

When one button is fully hidden, a flipper control should show up to allow the users to reveal the button again.

## 😯 Current Behavior

The flipper control will appear only after 3-4 buttons were hidden (vs. 1)

## 🔦 Context

My application has lots of buttons in a tool bar, and I would like to use horizontal-scroll to handle the navigation when buttons were hidden.

## 🌍 Your Environment

* OS & Device: MacOS
* Browser: Google Chrome
* Version: fast-components 2.16.2 & fast-element 1.6.2

**Comment by chrisdholt** [comment]
@SethDonohue can you investigate?

**Comment by SethDonohue** [comment]
@chrisdholt I have been digging in... 
 I cloned the node where the scroll stops are updated, and it looks to be a non-styled version of the item, instead of the final item with the correct width and height. This leads to all the scroll stops being off slightly (typically the padding is not included in the width used to calculate the scroll stops), leading to the last few items being outside of the total width, so it doesn't trigger the flippers to show up.

TLDR: Trying to sort how to update the scroll stops *after* the items are fully styled/rendered.
